rpc-jobs/RPC-AIO.yml

```diff
@@ -87,7 +87,7 @@
     concurrent: true
     properties:
       - build-discarder:
-          num-to-keep: 30
+          num-to-keep: 7
```

Comment:
You are right, I was thinking days.  However, 7 days is still going to be too much once we flip on per commit gating.  I need to rethink this a little.